### PR TITLE
adduser: add Indonesian translation

### PR DESCRIPTION
**Pull request by punkestu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/linux/adduser.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# adduser
+
+> Utilitas penambahan pengguna.
+> Informasi lebih lanjut: <https://manpages.debian.org/latest/adduser/adduser.html>.
+
+- Buat seorang pengguna baru dengan sebuah direktori home bawaan dan mendesak pengguna untuk mengatur sebuah password:
```
Comment (reinhart1010) [inline pages.id/linux/adduser.md:6]:
```suggestion
- Buat seorang pengguna baru dengan sebuah direktori pangkal/home bawaan dan mendesak pengguna untuk mengatur sebuah kata sandi:
```

The word `pangkal` is used to be consistent with [`cd`](https://github.com/tldr-pages/tldr/blob/main/pages.id/common/cd.md).

File: pages.id/linux/adduser.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`adduser {{nama_pengguna}}`
+
+- Buat seorang pengguna baru tanpa sebuah direktori home:
```
Comment (reinhart1010) [inline pages.id/linux/adduser.md:10]:
```suggestion
- Buat seorang pengguna baru tanpa sebuah direktori pangkal/home:
```

File: pages.id/linux/adduser.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`adduser --no-create-home {{nama_pengguna}}`
+
+- Buat seorang pengguna baru dengan sebuah direktori home di jalur yang telah dispesifikasikan:
```
Comment (reinhart1010) [inline pages.id/linux/adduser.md:14]:
```suggestion
- Buat seorang pengguna baru dengan sebuah direktori pangkal/home di jalur yang telah dispesifikasikan:
```

File: pages.id/linux/adduser.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`adduser --home {{jalur/ke/home}} {{nama_pengguna}}`
+
+- Buat seorang pengguna baru dengan shell yang telah dispesifikasikan sebagai shell login:
+
+`adduser --shell {{jalur/ke/shell}} {{nama_pengguna}}`
+
+- Buat seorang pengguna baru yang masuk ke grup yang dispesifikasikan:
```
Comment (reinhart1010) [inline pages.id/linux/adduser.md:22]:
```suggestion
- Buat seorang pengguna baru yang masuk ke grup pengguna yang dispesifikasikan:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for your contribution. Here are some of my notes to review.

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**